overlay/sbin/toolbox

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Licened under Apache 2.0
+# A simple way to bring your own tools to the K3OS Kubernetes Linux distribution
+# Inspired by https://github.com/containers/toolbox
+
+if [ $(whoami) != "root" ]
+then
+	echo "This script must be run as root."
+	exit 1
+fi
+
+if [ "$1" = "" ]; then
+	echo "toolbox: missing command" >&2
+	echo >&2
+	echo "These are some common commands:" >&2
+	echo "create	Create a new toolbox container" >&2
+	echo "enter	 Enter an existing toolbox container" >&2
+	echo "rm	 Remove an existing toolbox container" >&2
+	echo >&2
+	exit 1
+fi
+
+op=$1
+shift
+
+CTR='k3s ctr'
+registry="docker.io/library"
+image="ubuntu:19.10"
```

Comment:
This `${registry}`/`${image}` split isn't necessary. Something more like the below would be preferable:
```
: ${TOOLBOX_IMAGE:=docker.io/library/ubuntu:bionic}
```
This allows the user to override via the `TOOLBOX_IMAGE` envvar.

Additionally, thoughts on defaulting to the Ubuntu LTS codename (aka bionic) vs the explicit, non-LTS versions you have here? Defaulting to the userpsace from whence the kernel comes makes sense to me so long as a reasonable method of override is available.

Reply:
I was explicit about non-LTS because I personally want the latest stuff - but I'll just use the override. Both changes adopted.